src/Service.Tests/dab-config.CosmosDb_NoSql.json

```diff
@@ -69,6 +69,9 @@
             },
             {
               "action": "delete"
+            },
```

Comment:
do you need to introduce another action? Isnt a combination of create + update sufficient to determine the permissions for `patch`?